src/custom/constants/lists.ts

```diff
@@ -62,19 +65,19 @@ export const DEFAULT_LIST_OF_LISTS_BY_NETWORK: NetworkLists = {
   }),
   [ChainId.KOVAN]: buildNetworkDefaultLists({
     chainId: ChainId.KOVAN,
-    networkLists: []
+    networkLists: [COMPOUND_LIST]
   }),
   [ChainId.RINKEBY]: buildNetworkDefaultLists({
     chainId: ChainId.RINKEBY,
-    networkLists: [RINKEBY_LIST]
+    networkLists: [RINKEBY_LIST, COMPOUND_LIST]
   }),
   [ChainId.ROPSTEN]: buildNetworkDefaultLists({
     chainId: ChainId.ROPSTEN,
-    networkLists: []
+    networkLists: [COMPOUND_LIST]
   }),
   [ChainId.GÖRLI]: buildNetworkDefaultLists({
     chainId: ChainId.GÖRLI,
-    networkLists: []
+    networkLists: [COMPOUND_LIST]
```

Comment:
Not added on this PR, but I wonder: does it make sense to add ropsten and gorli at all, since we do not support them?

Reply:
probably not but i think the type requires it

Reply:
yes, it's required. I just added compound in case one day we support this network, it will be nice to have the tokens